haskell-build/haskell-build-orb.yml

```diff
@@ -107,6 +107,24 @@ examples:
                   - run: echo "I run after tests"
 
 executors:
+  ghc-7.10.3:
```

Comment:
Do all of these images contain `cabal >= 2.2` (or where `new-configure`/`new-build` + git support appear)?

Reply:
I believe all the images have the same version of cabal.

`7.10.3` definitely works with our images: https://circleci.com/workflow-run/6adfcf96-1473-4c83-aaf6-b4f52a915e11